UserManual/chapter_DataStructures.Rnw

```diff
@@ -333,7 +333,7 @@ eigenListFunctionGenerator <- nimbleFunction(
   },
   run = function(){
     ## we will take the eigendecomposition and svd of a symmetric matrix
-    eigenAndSvdList$demoEigenList <<- eigen(demoMatrix, only.values = TRUE)
+    eigenAndSvdList$demoEigenList <<- eigen(demoMatrix, symmetric = TRUE, only.values = TRUE)
```

Comment:
Consider moving `symmetric` to the third argument. It is generally safer to add parameters to the end of a parameter list, so as not to break existing user code with positional arguments like `eigen(X, FALSE)`.